.github/workflows/update-contributors.yaml

```diff
@@ -4,6 +4,9 @@ on:
   push:
     branches:
       - 'main'
+  schedule:
+  # Schedule at 9 AM JST (0 AM UTC) every Friday
+  - cron: '0 0 * * 5'
```

Comment:
GitHub Actions' scheduled workflows are configured to run on UTC time. Currently, it's not possible to set a timezone directly in the cron syntax.

FYI: 
https://github.com/orgs/community/discussions/13454